tests/fixture/pageObject/playwright/PlaywrightHelper.ts

```diff
@@ -12,16 +13,17 @@ export class PlaywrightHelper {
     private readonly page: Page;
     private readonly polly: PollyObject;
     private readonly server: ServerSideRender;
+    private readonly storeUrl: string;
     private isDevMode: boolean = false;
     private har: string = '';
-    private storeUrl: string = '';
 
     constructor(page: Page) {
         this.page = page;
         this.mode = process.env.MODE?.toLowerCase() === 'record'  ? 'record' : 'replay';
         this.isReplay = this.mode === 'replay';
         this.polly = new PollyObject(this.mode);
         this.server = new ServerSideRender();
+        this.storeUrl = (this.isReplay) ? '' : getStoreUrl();
```

Comment:
Probably don't need this conditional.

Reply:
I have removed this control flow here and made `getStoreUrl()` mode-aware. 
As a result, it will complain about `STOREURL` undefined in `record` mode only.